examples/blender/blender.py

```diff
@@ -73,9 +73,12 @@ async def worker(ctx: WorkContext, tasks):
 
     # iterator over the frame indices that we want to render
     frames: range = range(0, 60, 10)
-    # TODO make this dynamic, e.g. depending on the size of files to transfer
+    # TODO make this more dynamic, e.g. depending on the size of files to transfer
```

Comment:
I think "this" in this sentence refers to `init_overhead`, which is not dynamic at all (so no need to add "more"). Anyway, if we're going to add/rephrase/change meaning of the comments I'd like this to be reviewed by @shadeofblue as well.